openreview/conference/templates/programchairWebfield.js

```diff
@@ -2967,13 +2967,27 @@ var buildCSV = function(){
   var areachairIds = conferenceStatusData.areaChairGroups.byNotes;
   var isFiltered = conferenceStatusData.filteredRows ? true : false
   var notes = isFiltered ? conferenceStatusData.filteredRows : conferenceStatusData.blindedNotes
-
+  var originalNotes = notes.map(p => isFiltered
+    ? p.note.details.original || p.note
+    : p.details.original || p
+  )
+  
+  var noteContentFields = [
```

Comment:
@xkopenreview can you create a utils function in webfield.js to get all the content fields or to exports the notes into an array of string? having the code in webfield.js you can use ES6 features

Reply:
ok. for your info the changes are generated by babel with target "since 2017"

Reply:
discussed with Melisa about this and decided not to have a util function in webfield.js as where the util function can be reused is not clear